www/src/pages/en/folder-structure.md

```diff
@@ -0,0 +1,281 @@
+├─ tailwind.config.cjs
+├─ tsconfig.json
+```
+
+## `prisma`
+
+The `prisma` folder contains the `schema.prisma` file which is used to configure the database connection and the database schema. It is also the location to store migration files and/or seed scripts, if used.
+
+<sub>(With Prisma)</sub>
+
+## `public`
+
+The `public` folder contains static assets that are served by the web server. The `favicon.ico` file is an example of a static asset.
+
+## `src`
+
+The `src` folder contains all the source code of the application.
+
+## `src/env`
+
+Used for environment variable validation and type definitions - see [Environment Variables](usage/env-variables).
+
+### `src/env/client.mjs`
+
+[See the client.mjs documentation](usage/env-variables#servermjs--clientmjs)
+
+### `src/env/server.mjs`
+
+[See the server.mjs documentation](usage/env-variables#servermjs--clientmjs)
+
+### `src/env/schema.mjs`
+
+[See the schema.mjs documentation](usage/env-variables#schemamjs)
+
+## `src/pages`
+
+The `pages` folder contains all the pages of the Next.js application. The `index.tsx` file at the root directory of `/pages` is the homepage of the application. The `_app.tsx` file is used to wrap the application with providers. See [Next.js documentation](https://nextjs.org/docs/basic-features/pages) for more information.
+
+## `src/pages/api`
+
+The `api` folder contains all the API routes of the Next.js application. The `examples.ts` file (with Prisma) contains an example of a route that uses the [Next.js API route](https://nextjs.org/docs/api-routes/introduction) feature along with Prisma. The `restricted.ts` file (with Next-Auth) contains an example of a route that uses the [Next.js API route](https://nextjs.org/docs/api-routes/introduction) feature and is protected by [NextAuth.js](https://next-auth.js.org/).
+
+<sub>(With NextAuth.js, tRPC or tRPC + Prisma)</sub>
+
+## `src/pages/api/auth`
+
+The `auth` folder contains the NextAuth.js authentication slug route.
+
+<sub>(With NextAuth.js)</sub>
+
+### `src/pages/api/auth/[...nextauth].ts`
+
+The `[...nextauth].ts` file is the NextAuth.js authentication slug route. It is used to handle authentication requests. See [NextAuth.js usage](usage/next-auth) for more information on NextAuth.js, and [Next.js Dynamic Routes Docs](https://nextjs.org/docs/routing/dynamic-routes) for info on catch-all/slug routes.
+
+<sub>(with NextAuth.js)</sub>
+
+## `src/pages/api/trpc`
+
+The `trpc` folder contains the tRPC API entry point.
+
+<sub>(with tRPC)</sub>
+
+### `src/pages/api/trpc/[trpc].ts`
+
+The `[trpc].ts` file is the tRPC API entry point. It is used to handle tRPC requests. See [tRPC usage](usage/trpc#-pagesapitrpctrpcts) for more information on this file, and [Next.js Dynamic Routes Docs](https://nextjs.org/docs/routing/dynamic-routes) for info on catch-all/slug routes.
+
+<sub>(with tRPC)</sub>
+
+## `src/server`
+
+The `server` folder is used to clearly seperate server-side code from client-side code.
```

Comment:
```suggestion
The `server` folder is used to clearly separate server-side code from client-side code.
```

Reply:
Fix in https://github.com/t3-oss/create-t3-app/pull/731/commits/8c792e93c2cba989c5f9d632de0168b0cec2e5d1, thanks for that 😅